### DISCUSSION TITLE
Welcome to Nix Discussions!

### DISCUSSION THREAD

**Discussion by SteveLauC** [nix-rust/nix#2445] [Announcements]
## 👋 Welcome to Nix discussions!

Issues are mostly for bugs and feature requests, if you have some questions or some random thoughts about Nix, we can discuss them here!

**Comment by Tools4m3** [comment]
Good day Folks,

Due to upgrades with OpenAI we cannot longer interact with ChatGPT using Safari Version 15.6.1 (15613.3.9.1.16, 15613) and MacOS version 10.15.7

We can work with it using Chrome or Firefox, but due to security reasons, we won't use any other browser to 
use/access this platform. 

Have you worked/developed a patch or a extension to work this out? Enclosed to this message you will find the error we encountered. 

Your help will be greatly appreciated.
![ChatGPT Interaction with Mac10 15](https://github.com/nix-rust/nix/assets/144577159/f406f5db-04d7-4c3c-8f92-58bd41f669d5)

**Comment by asomers** [comment]
Closing as off-topic.

**Comment by SteveLauC** [comment]
Looks like my account has been marked spammy due to this discussion, and I have been removed from contributor list:DDD
